src/OpenLoco/src/Input/Keyboard.cpp

```diff
@@ -589,5 +592,21 @@ namespace OpenLoco::Input
         }
 
         keyScroll();
+
```

Comment:
put this above keyScroll to sink input. Also fix the edge case where shift and control flip on the same frame.